benchmarks/d-adreno.json

```diff
@@ -1 +1 @@
-["3.0.0",[" qualcomm adreno 540","540",0,[[1919,1279,19]]],["qualcomm qualcomm qualcomm� adreno� 540 gpu","540",0,[[1919,1279,25]]],["qualcomm qualcomm qualcommr adreno? 540 gpu","540",0,[[1919,1080,25]]],["qualcomm qualcomm qualcomm adreno 630 gpu","630",0,[[1920,1080,21]]],["qualcomm qualcomm qualcomm adreno 540 gpu","540",0,[[1919,1279,24]]],["qualcomm qualcomm qualcomm adreno 680 gpu","680",0,[[3840,2160,25]]]]
\ No newline at end of file
+["3.0.0",["qualcomm adreno 540","540",0,[[1919,1279,19]]],["qualcomm adreno 630 gpu","630",0,[[1920,1080,21]]],["qualcomm adreno 540 gpu","540",0,[[1919,1279,24]]],["qualcomm adreno 680 gpu","680",0,[[3840,2160,25]]]]
```

Comment:
Could you update the package version to 3.0.1 (as it is a patch), that way we keep the updated benchmarks in line with the new release.